config/tips.txt

```diff
@@ -0,0 +1,136 @@
+The best antagonist rounds usually tend to include most players in someway.
+Losing is !FUN!, each round lost is a lesson to learn and apply for the next round.
+Changelings can take peoples voices, appearance and memories.
+Bluespace Revenants will slowly distort overtime without creating runes and feeding on blood.
+Fire Extingushers can be used to wet floors to slip people with.
+You can remove fingerprints on objects by using a rag on them.
+You may leave partial fingerprints with specific types of gloves.
+You may leave fibers of identifiable clothing on any object you interact with, swapping to less identifable clothing is always wise before doing something bad.
+Flashers can repeatidly stun cyborgs and disable their movement and tools.
+Emagging a cyborg only unlocks their access panel, and does not upload custom laws.
+You can redirect package-wrapped items using the delivery tagger and deliver them to a specific area.
+Items can be thrown down Z-levels to unsuspecting people below.
+Screwdrivers allow you to change the timers on a grenade to detonate at different times.
+Insulated Gloves can be redyed to another colour using a crayon in the washing machine.
+Budget Insulated Gloves will either reduce or multiply the amount of electrical shock damage you get.
+Leaving the Emitter on in the Engine Room is a unwise decision.
+Emagging the Fax Machine lets you send messages to syndicate contacts.
+Praying to the gods may grant you all sorts of boons, or just a cookie.
+The best judgements are always made on the decision of a magic 8-ball.
+It is good practice to remove a victim's headset and turn-off their suit sensors while they are still alive, to avoid medbay or security noticing.
+You can disguise yourself to look like a cyborg by wearing a cardboard helmet and cardboard suit and can control what sprite dependant on your backpack.
+Gunshots and cleaned blood will leave residue which can be found using the Detective's luminol sprayer.
+You can put your PDA in your ID slot and insert you ID into it in order to save space.
+You can delete emails sent to you on your PDA, but they will still be visible on the PDA Messanger Server in Engineering.
+The camera is the AI's eyes, if the AI is rogue, it is wise to disable the camera nearby to prevent them for seeing you.
+You can replace the pen in your PDA with other pens, such as para-pens.
+You can replace walls with false-walls, these walls however will not carry over any paint applied above it.
+You can fill bottles with welding fuel and stuff them with a rag to create a molotov.
+The Bluespace Drive is destructible and can create horrible anomalies if destroyed when the ship is jumping.
+You can respawn back into the round after 10 minutes, but remember to spawn as a different character with the memories of what has occured already in the round.
```

Comment:
Some odd wording here. You _can_ join as a character with memories that have happened in the round, so long as they cryod and weren't removed. 
```suggestion
You can respawn back into the round after 10 minutes, but remember to spawn as a different character which has not already died in the round. You must not use the memories of your dead characters to your advantage, doing so will result in staff action
```